workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Model/Details/UnityComponentDetails.cs

```diff
@@ -12,8 +13,8 @@ public class UnityComponentDetails
         public bool IsBlittable { get; }
 
         public IReadOnlyList<UnityFieldDetails> FieldDetails { get; private set; }
-        public IReadOnlyList<UnityCommandDetails> CommandDetails { get; }
-        public IReadOnlyList<UnityEventDetails> EventDetails { get; }
+        public IReadOnlyList<UnityCommandDetails> CommandDetails { get; private set; }
```

Comment:
Why are these 2 private set now?

Reply:
nice spot - these are relics from when the filtering was done in a separate method